File: pythonstudy_day12/appliance/Audio.py
from appliance.MediaAppliance import MediaAppliance


class Audio(MediaAppliance):
    def __init__(self, power, volumn):
        super().__init__(power, volumn)
        
    def tune(self):
        str = "hi" if self.power else "turn it on"
        print(str)
        

File: day12/src/com/inheritance/Audio.java
package com.inheritance;
import java.io.*;
import java.util.*;

/**
 * 
 */
public class Audio extends Media {

    /**
     * Default constructor
     */
    public Audio() {
    	super();
    }
    
    public Audio(boolean power, int volumn) {
    	super(power, volumn);
    }

    public void tune() {
        // TODO implement here
    	String str = (this.isPower()) ? "hi":"turn it on";
    	System.out.println(str);
    }

}